src/components/map/layers/NodeLayer.tsx

```diff
@@ -17,37 +17,82 @@ interface MarkerLayerProps {
     sidebarStore?: SidebarStore;
 }
 
-enum color {
-    STOP_BORDER_COLOR = '#f17c44',
-    CROSSROAD_BORDER_COLOR = '#666666',
-    NORMAL_FILL_COLOR = '#FFF',
-    SELECTED_FILL_COLOR = '#f44268',
-}
```

Comment:
Any reason why removing these and setting the color codes directly in the code? I think this old style to specify colors was rather ok.